### PR TITLE
[crow] update to 1.0

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: ports/crow/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,24 @@
 {
   "name": "crow",
-  "version": "0.3.4",
+  "version": "1.0.0",
```
Comment (ras0219-msft) [inline ports/crow/vcpkg.json:3]:
```suggestion
  "version": "1.0-1",
```

**Comment by autoantwort** [comment]
@JonLiu1993 ping

File: versions/c-/crow.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "e21a4fb8fd849bfe3ee8d77dbdb3a4953dd08d55",
+      "version": "1.0-1",
+      "port-version": 0
+    },
+    {
+      "git-tree": "2816b6ed467bc82728212fa4c4190512ca3e6b26",
+      "version": "1.0.0",
+      "port-version": 0
+    },
```
Comment (JonLiu1993) [inline versions/c-/crow.json:12]:
```suggestion
```

Comment (autoantwort) [reply versions/c-/crow.json:12]: Ah I ses. Thanks :)

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**